### DISCUSSION TITLE
any way to change path?

### DISCUSSION THREAD

**Discussion by end3r-man** [inertiajs/inertia#1979] [Help]
is their any way to change the default path from Pages to views/

**Comment by pedroborges** [comment]
You are free to change the `import.meta.glob` in your `app.js` file:

```diff
- const pages = import.meta.glob('./Pages/**/*.vue', { eager: true })
+ const pages = import.meta.glob('./views/**/*.vue', { eager: true })
```

If you are testing your app in Laravel, you will need to update the `inertia.testing.page_paths` array in the `config/inertia.php` file. Publish this config file if you haven't already.
